launch.go

```diff
@@ -587,9 +592,16 @@ func main() {
 
 		buildToken, err := temporalApi.GetBuildToken(buildID, c.Int("build-timeout"))
 		if err != nil {
+			// If token is build scope token, GetBuildToken returns 403 error but need not exit
 			log.Printf("Error getting Build Token %v: %v", buildID, err)
 		}
 
+		if fetchFlag {
```

Comment:
we shouldn't be fetching `buildToken` if `fetchFlag` is not passed.

Reply:
I changed the scope of fetchFlag if statement to only fetch `buildToken` if `fetchFlag` is passed.